### ISSUE TITLE
No Capture when using port 5400 for DNS

### ISSUE THREAD

**Issue by vietde** [AdguardTeam/AdGuardHome#2210]
### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [ *] I am running the latest version
- [ *] I checked the documentation and found no answer
- [* ] I checked to make sure that this issue has not already been filed

### Issue Details

* **Version of AdGuard Home server:**
  *  0.103.3
* **How did you setup DNS configuration:**
  *  IoT
* **If it's a router or IoT, please write device model:**
  *  Raspberry Pi 1
* **Operating system and version:**
  *  Raspbian release 10

### Screenshots

<details><summary>Screenshot:</summary>

I'm using DDWRT router and change my dns port to :5400; however, Adguard still does not not capture any log. Port 5400 is listening but not capture.

</details>

**Comment by ameshkov** [comment]
I don't understand what you're trying to achieve and how have you setup your devics to use AGH

**Comment by vietde** [comment]
> 
> 
> I don't understand what you're trying to achieve and how have you setup your devics to use AGH

What I try to say is I have an old RPi to use as DNS server. I configured my router DNS to point to my rpi as instructed. Some reason my AdGuardHome doesn't capture any thing. I tried to change dns port still the same. Turns out it only listen IPv6 which I don't configure. Any ideas? 

> pi@homepi:~/AdGuardHome $ sudo lsof -i:5053
> COMMAND    PID USER   FD   TYPE DEVICE SIZE/OFF NODE NAME
> AdGuardHo 6608 root   16u  IPv6  40694      0t0  UDP *:5053
> AdGuardHo 6608 root   17u  IPv6  40695      0t0  TCP *:5053 (LISTEN)


`pi@homepi:~/AdGuardHome $ sudo netstat -ltp 5053
Active Internet connections (only servers)
Proto Recv-Q Send-Q Local Address           Foreign Address         State       PID/Program name
tcp        0      0 localhost:6010          0.0.0.0:*               LISTEN      712/sshd: pi@pts/0
tcp        0      0 0.0.0.0:5900            0.0.0.0:*               LISTEN      378/vncserver-x11-c
tcp        0      0 0.0.0.0:http            0.0.0.0:*               LISTEN      420/lighttpd
tcp        0      0 0.0.0.0:ssh             0.0.0.0:*               LISTEN      392/sshd
tcp6       0      0 localhost:6010          [::]:*                  LISTEN      712/sshd: pi@pts/0
tcp6       0      0 [::]:5053               [::]:*                  LISTEN      6608/AdGuardHome
tcp6       0      0 [::]:5900               [::]:*                  LISTEN      378/vncserver-x11-c
tcp6       0      0 [::]:http-alt           [::]:*                  LISTEN      6608/AdGuardHome
tcp6       0      0 [::]:http               [::]:*                  LISTEN      420/lighttpd
tcp6       0      0 [::]:ssh                [::]:*                  LISTEN      392/sshd
`
